lib/Serialization/Serialization.cpp

```diff
@@ -4809,6 +4929,7 @@ void Serializer::writeAST(ModuleOrSourceFile DC,
     writeOffsets(Offsets, SubstitutionMapsToSerialize);
     writeOffsets(Offsets, NormalConformancesToSerialize);
     writeOffsets(Offsets, SILLayoutsToSerialize);
+    writeOffsets(Offsets, ClangTypesToSerialize);
```

Comment:
Could you move this line to earlier, right after `writeOffsets(Offsets, TypesToSerialize);`? That way the order matches the order for `wroteSomething |= blah`.

Reply:
Sure